util/test/testUtil.h

```diff
@@ -44,7 +45,7 @@ void valuecheck(const Eigen::DenseBase<DerivedA>& a, const Eigen::DenseBase<Deri
 
 void valuecheck(double a, double b, double tolerance = 1e-8)
 {
-  if (abs(a - b) > tolerance) {
+  if (std::abs(a - b) > tolerance) {
```

Comment:
so drakeUtil is currently more of a drakeMexUtil (apart from some methods that don't look like they belong anymore). How do you feel about including mex stuff in source files that would otherwise not need it?